### PR TITLE
https://dev.azure.com/cyrovalente/HNI%20Modernization/_workitems/edit…

### PR DESCRIPTION
**Pull request by weskubo-cgi**
…/3145. Update HNClient to expect fhir+json.

### CODE REVIEW

File: hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/HnsClientMainMethod.java [hunk 1]
```diff
@@ -5,7 +5,7 @@
 /**
  * Main class that boots the Camel application.
  */
-public class MainMethod {
+public class HnsClientMainMethod {
```
Comment (daveb-hni) [inline hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/HnsClientMainMethod.java:8]: Need to update the class name in the camel-maven-plugin in the pom also.

File: hnclient-v2/src/main/resources/application.properties [hunk 2]
```diff
@@ -14,7 +14,7 @@ max-socket-read-tries=100
 # For HNSecure:
 http-protocol=http
 hnsecure-hostname = localhost
-hnsecure-port = 14885
+hnsecure-port = 14890
```
Comment (daveb-hni) [inline hnclient-v2/src/main/resources/application.properties]: Probably no need to commit this, only changed locally for testing.

File: hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/HnsClientMainMethod.java (hunk 1 above)
Comment (weskubo-cgi) [reply hnclient-v2/src/main/java/ca/bc/gov/hlth/hnclientv2/HnsClientMainMethod.java:8]: Fixed in ce24d4153daacf73c6f52ccfbfec86da003b65ca.

File: hnclient-v2/src/main/resources/application.properties (hunk 2 above)
Comment (weskubo-cgi) [reply hnclient-v2/src/main/resources/application.properties]: Reverted port in ce24d4153daacf73c6f52ccfbfec86da003b65ca.

**Pull request merged**